packages/docs/pages/icons.tsx

```diff
@@ -35,162 +35,193 @@ const IconsPage = () => {
-                        ]}
-                        placeholder={'Choose country'}
-                        placement={'bottom-start'}
-                        required
-                        value={value}
-                      />
-                    </FormGroup>
-                  </Form>
-                );
-              }}
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-        </>
-      ),
-    },
-    {
-      id: 'props',
-      title: 'Props',
-      render: () => (
-        <>
-          <IconPropTable>
-            <CodeSnippet showControls={false} language="bash">
-              npm install @bigcommerce/big-design-icons // or yarn add @bigcommerce/big-design-icons
-            </CodeSnippet>
-          </IconPropTable>
-          <FlagIconPropTable>
-            <InlineMessage
-              type="warning"
-              messages={[
-                {
-                  text: 'Flags are a submodule of the Icons package "@bigcommerce/big-design-icons", you will need to add the package first using "yarn add @bigcommerce/big-design-icons".',
-                },
-              ]}
-              marginBottom="medium"
-            />
-          </FlagIconPropTable>
-        </>
-      ),
-    },
-    {
-      id: 'icons',
-      title: 'Available icons',
-      render: () => (
-        <Panel header="Available icons">
-          <Flex flexWrap="wrap" justifyContent="center">
-            {Object.keys(icons).map((iconName) => {
-              const Icon = icons[iconName];
-
-              return (
-                <Flex
-                  key={iconName}
-                  style={{ width: '300px' }}
-                  flexDirection="column"
-                  borderRadius="normal"
-                  justifyContent="center"
-                  alignItems="center"
-                  padding="large"
-                >
-                  <Icon size="xxxLarge" />
-                  <Code>{`<${iconName} />`}</Code>
-                </Flex>
-              );
-            })}
-          </Flex>
-        </Panel>
-      ),
-    },
-  ];
-
   return (
     <>
       <H1>Icon</H1>
 
-      <PageNavigation items={items} />
+      <Panel header="Overview" headerId="overview">
+        <Text>
+          BigDesign offers a variety of <NextLink href={{ hash: 'available-icons' }}>icon</NextLink> components that can
+          be used in a project. We also provide{' '}
+          <NextLink href={{ query: 'implementation=country-flags' }}>Country flags</NextLink> icons that can be
+          imported.
+        </Text>
+        <Text>
+          To use the icons install <Code primary>@bigcommerce/big-design-icons</Code> package from npm.
+        </Text>
+
+        <Text bold>When to use:</Text>
+        <List>
+          <List.Item>To represent an action or a status.</List.Item>
+          <List.Item>To provide additional visual cues.</List.Item>
+        </List>
+      </Panel>
```

Comment:
Note: I added this overview and when to use section,  please feel free to make suggestions. Thanks!